pom.xml

```diff
@@ -12,7 +12,7 @@
   <groupId>org.kie</groupId>
   <artifactId>kie-gwthelper-maven-plugin</artifactId>
   <packaging>maven-plugin</packaging>
-  <version>1.0</version>
+  <version>1.1-SNAPSHOT</version>
```

Comment:
@mareknovotny 
Due to some enforcer rule we can not have SNAPSHOT versions on plugins

Reply:
yes, no problem. We should have referenced in other repositories only the released versions

So the master is development stable branch, and the released version is used in other kiegroup repositories.

Reply:
just to be sure the change to 1.1 version needs to happen in release of 1.1 by tagging that version and release it into Maven Central. Then you can update the version in other repository